### PR TITLE
Fixed ignore_time property not found during upgrade

### PR DESCRIPTION
**Pull request by MiguelazoDS**
|Related issue|
|---|
|#8062|
## Description
As consequence of the change introduced in #7892, after an upgrade Wazuh was no able to start, due to the `ignore_time` property in the ossec.conf file is not found in vuln_det module. 

## DoD
Warning that shows that the property is deprecated. 

image
## Tests

- Compilation without warnings in every supported platform
  - [X] Linux
- [X] Source installation
- [X] Source upgrade
- [X] Review logs syntax and correct language

### CODE REVIEW

**Review by palaciosjeremias — APPROVED** [review]
LGTM

**Review by pereyra-m — APPROVED** [review]
LGTM!

File: src/config/wmodules-vuln-detector.c [hunk 1]
```diff
@@ -416,6 +417,8 @@ int Read_Vuln(const OS_XML *xml, xml_node **nodes, void *d1, char d2) {
                 merror("Invalid content for tag '%s' at module '%s'", XML_RUN_ON_START, WM_VULNDETECTOR_CONTEXT.name);
                 return OS_INVALID;
             }
+        } else if (!strcmp(nodes[i]->element, XML_IGNORE_TIME)){
+            mwarn("Detected deprecated tag <%s> at module '%s'.", nodes[i]->element, WM_VULNDETECTOR_CONTEXT.name);
```
Comment (DProvinciani) [inline src/config/wmodules-vuln-detector.c:421]: As per the discussion that we had, change this log for something like:

```
The <%s> tag at module '%s' is deprecated for version newer than 4.3.
```

**Review by DProvinciani — APPROVED** [review]
Looks good!

File: src/config/wmodules-vuln-detector.c (hunk 1 above)
Comment (MiguelazoDS) [reply src/config/wmodules-vuln-detector.c]: Done!

**Pull request merged**